indico/modules/events/contributions/models/contributions.py

```diff
@@ -242,6 +242,10 @@ def protection_parent(self):
     def track(self):
         return self.event_new.as_legacy.getTrackById(str(self.track_id))
 
+    @property
+    def is_scheduled(self):
+        return self.timetable_entry and self.timetable_entry.start_dt is not None
```

Comment:
`return self.timetable_entry is not None` is enough. `start_dt` is not nullable.